components/CustomTextfield/Contact.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+import CustomButton from '@/components/CustomButton'
+
+function Contact(): React.JSX.Element {
+  return (
+    <div>
+      <form className="flex h-[624.04px] w-[514.56px] flex-col rounded-[10px] bg-white p-12">
```

Comment:
Could u change `h-[624.04px] w-[514.56px] rounded-[10px]` ?  We should use values in tailwind predefined classes.